src/Reservoir.jl

```diff
@@ -32,27 +40,50 @@ function run_timestep(c::Reservoir, tt::Int)
     v = c.Variables
     p = c.Parameters
     d = c.Dimensions
-    if tt==1
-        for rr in d.reservoirs
-            v.releases[rr, tt] = p.inflows[rr,tt] - p.captures[rr, tt]
-            v.storage[rr,tt] = (1-p.evaporation[rr,tt])*p.storage0[rr] + p.captures[rr, tt]
+
+    v.inflows[:,tt] = zeros(numreservoirs); 
+    v.outflows[:,tt] = zeros(numreservoirs); 
+    rr = 1
+    for gg in 1:numgauges
+       index = vertex_index(downstreamorder[gg])
+       if isreservoir[index] > 0
+	     v.inflows[rr,tt] = p.inflowsgauges[gg, tt]*1000; 
+	     v.outflows[rr,tt] = p.outflowsgauges[gg, tt]*1000; 
+	     rr += 1
```

Comment:
Fabulous ! I am always a bit confused when it comes to gauges-reservoirs-counties connections ... I'll make the changes